functions/tsconfig.json

```diff
@@ -1,17 +1,26 @@
 {
+  "files": ["./__tests__/test.types.d.ts"],
   "compilerOptions": {
+    "skipLibCheck": true,
+    "lib": ["esnext.asynciterable"],
```

Comment:
Why this lib in particular ?

Reply:
I needed it at some point I just forgot to remove it.